what_is.md

```diff
@@ -273,32 +306,61 @@ The trees reflect, for example, the origin and age of alleles under
 selection, the spatial structure of populations, and the effects
 of hybridization and admixture in the past.
 
-```{todo}
-Insert illustration of the above, perhaps a demesdraw plot, linking to the tree sequence
-in the next code box.
-```
+The tree sequence in the tutorial was actually generated using a model of population
+splits and expansions as shown in the following schematic, plotted using the fantastic
```

Comment:
"fantastic" might be bit over excited, can probably delete

Reply:
:) sure. There's a similarly enthusiastic "Thanks to the excellent scikit-allel for the basic plotting code used here" in the stats tut though. Other adjective suggestions welcome.

Reply:
As I said, I would delete

Reply:
Yes, I will. I mean, should I also delete "excellent" referring to scikit-allel from the stats tut?